benchmarks/dynamo/check_graph_breaks.py

```diff
@@ -6,17 +6,20 @@
 import pandas as pd
 
 
-def get_field(csv, model_name: str, field: str, typ=float):
-    return typ(csv.loc[csv["name"] == model_name][field])
+def get_field(csv, model_name: str, field: str):
+    try:
+        return csv.loc[csv["name"] == model_name][field].item()
+    except Exception as e:
+        return None
```

Comment:
don't suppress all exceptions. Is there a more specific exception type you can catch here?